### PR TITLE
Telco 92 integration tests

### PR DESCRIPTION
**Pull request by Gmerold**

### CODE REVIEW

File: .github/workflows/orchestrator.nms.magmalte.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    paths:
+    - 'orchestrator-bundle/nms-magmalte-operator/**'
```
Comment (sanchezfdezjavier) [inline .github/workflows/orchestrator.nms.magmalte.yml]: Shouldn't the `-` be intended?
```suggestion
    paths:
      - 'orchestrator-bundle/nms-magmalte-operator/**'
```

Comment (sanchezfdezjavier) [reply .github/workflows/orchestrator.nms.magmalte.yml]: Actually, I think this applies to a lot of files. Idk know if this thing would throw a YAML error, but might be worth checking

Comment (Gmerold) [reply .github/workflows/orchestrator.nms.magmalte.yml]: You're right, over half of the yamls had that wrong... yet worked somehow... magic :D
Anyway, according to the docs, it should be indented.
Fixed.

File: orchestrator-bundle/nms-magmalte-operator/tests/integration/test_integration.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
+class TestNmsMagmaLTE:
+    @pytest.fixture(scope="module")
+    async def setup(self, ops_test):
+        await self._deploy_postgresql(ops_test)
+        await self._deploy_orc8r_certifier(ops_test)
+
+    @pytest.mark.abort_on_fail
+    async def test_build_and_deploy(self, ops_test, setup):
```
Comment (sanchezfdezjavier) [inline orchestrator-bundle/nms-magmalte-operator/tests/integration/test_integration.py:29]: There's no reference to the `setup` parameter within the function. Should it be dropped?

Comment (Gmerold) [reply orchestrator-bundle/nms-magmalte-operator/tests/integration/test_integration.py:29]: It's not a parameter; it's a reference to a fixture (defined above).

Comment (sanchezfdezjavier) [reply orchestrator-bundle/nms-magmalte-operator/tests/integration/test_integration.py:29]: This is repeated in more `test_integration.py` files

**Review by sanchezfdezjavier — APPROVED** [review]

**Pull request merged**